test/index.spec.js

```diff
@@ -0,0 +1,11 @@
+const heml = require('../packages/heml')
+const fixtures = require('./__fixtures__')
+
+describe('snapshots', () => {
+  for (const filename in fixtures) {
+    const simpleName = filename.replace(/\.fixture\.js$/, '')
+    const fixture = fixtures[filename]
+
+    test(simpleName, () => heml(fixture).then((result) => expect(result).toMatchSnapshot()))
```

Comment:
most of the code base is written with async await, I could reformat the test to use that if you prefer.